<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionWizard</class>
 <widget class="QWizard" name="PrescriptionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Prescription Wizard</string>
  </property>
  <widget class="Prescription_00" name="page0">
   <property name="title">
    <string>Medication Search</string>
   </property>
   <property name="subTitle">
    <string>Select the medication.</string>
   </property>
  </widget>
  <widget class="Prescription_01" name="page1">
   <property name="title">
    <string>Prescription Information</string>
   </property>
   <property name="subTitle">
    <string>Enter the prescription details.</string>
   </property>
  </widget>
  <widget class="Prescription_02" name="page2">
   <property name="title">
    <string>Instructions</string>
   </property>
   <property name="subTitle">
    <string>Enter the prescription instructions.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Prescription_00</class>
   <extends>QWizardPage</extends>
   <header location="global">prescription_00.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Prescription_01</class>
   <extends>QWizardPage</extends>
   <header location="global">prescription_01.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Prescription_02</class>
   <extends>QWizardPage</extends>
   <header location="global">prescription_02.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
